extensions/hotkey/init.lua

```diff
@@ -143,6 +143,19 @@ local function getIndex(mods,keycode) -- key for hotkeys table
   key=key and supper(key) or '[#'..keycode..']'
   return mods..key
 end
+
+local function getFunc(f)
+  if f == nil then return nil end
+  if type(f) == 'function' then return f end
+  if type(f) == 'table' then
+    local m = getmetatable(f)
+    if m and m.__call then
+      return getFunc(m.__call)
```

Comment:
This is a very edge case, but this is potentially infinitely recursive if someone has a weird/broken table. How about just changing line 152/153 to:

```lua
if m and m.__call and type(m.__call) == 'function' then
  return m.__call
```

?